packages/typescript/ai-openrouter/tests/openrouter-adapter.test.ts

```
@@ -1,16 +1,24 @@
 import { beforeEach, describe, expect, it, vi } from 'vitest'
 import { chat } from '@tanstack/ai'
+import { HTTPClient } from '@openrouter/sdk'
 import { ChatRequest$outboundSchema } from '@openrouter/sdk/models'
 import { createOpenRouterText } from '../src/adapters/text'
+import type { OpenRouterTextAdapter } from '../src/adapters/text'
```

ESLint

[error] '../src/adapters/text' imported multiple times.

(import/no-duplicates)